client/src/views/sidebar-menu/SidebarMenuPage.vue

```diff
@@ -344,12 +342,7 @@ function onEnd(): void {
 }
 
 function resizeMenu(newWidth: number): void {
-  if (splitPane.value) {
-    splitPane.value.$el.style.setProperty('--side-width', `${newWidth}px`);
-  }
-  if (divider.value) {
-    divider.value.style.setProperty('left', `${newWidth - 4}px`);
-  }
+  document.documentElement.style.setProperty('--parsec-sidebar-menu-width', `${newWidth}px`);
```

Comment:
Following the previous comment, we might need a function to convert px to rem.

Reply:
see previous